src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -87,9 +87,9 @@ const leftBorderRadius = `${endRadii} 0px 0px ${endRadii}`;
 const rightBorderRadius = `0px ${endRadii} ${endRadii} 0px`;
 const centerBorderRadius = "var(--click-button-button-group-radii-center)";
 
-const Button = styled.button.attrs<ButtonProps>(
```

Comment:
Sorry, a few of these are my prettier settings that auto format on save, couldn't figure out how to turn it off though.